source/code/utilities/chinese/chinese_translations.hcp

```diff
@@ -677,16 +707,16 @@ class
         Translation_Entry x;
         x.english = L"How can you not go to class?";
         x.pinyin = L"nǐ zěn me bù qù shàng kè?";
-        x.chinese = L"你怎么不去上课？";
+        x.chinese = L"你怎麼不去上課？";
```

Comment:
wow, I wonder why all of these were inaccurate.  can't remember if I took them directly from a site or did a google translate.  Or are you converting them from simplified to traditional?